app/views/layouts/form_only.html.haml

```diff
@@ -20,10 +20,11 @@
 
   %body#form-only{ class: I18n.locale }
     %header#top
-      %a{ href: root_path }
-        %img(src="/assets/header/logo400-blue@2x.png" width="400" height="49" alt="Energy Transition Model")
+      .container
```

Comment:
Personally, I think I preferred the logo center-aligned. It doesn't look quite on the left. The gears in the 400px logo are slightly too wide, which causes "Energy Transition Model" to be misaligned with the content inside the "Create your account" container.